src/plugins/europeana/record.js

```diff
@@ -301,8 +312,11 @@ export default (context = {}) => {
     * @return {Boolean} true if enriched data will be shown
     */
     localeSpecificFieldValueIsFromEnrichment(field, providerProxy, proxies, predictedUiLang) {
-      if (providerProxy[field][predictedUiLang] ||
-        (!proxies[2][field][predictedUiLang] && providerProxy[field]['en'])) {
+      if (isLangMap(providerProxy[field]) &&
```

Comment:
Let's move this check to its own function to reduce cognitive complexity.

Reply:
I've pushed some refactor, hopefully making this a bit clearer.